src/components/showtask.vue

```diff
@@ -3,17 +3,34 @@
     <div 
       class="department-div" 
       v-show="!checked">
-      <input 
-        type="checkbox" 
-        class="checkbox-check" 
-        v-model="checked" 
-        @change="check($event)">
-      <p class="task-title">{{task.title}}</p>
-      <p class="task-author">{{task.author}}</p>
-      <p class="task-date">{{task.date}}</p>
+      <div class="mdc-form-field">
+        <div class="mdc-checkbox">
+          <input type="checkbox"
+            v-model="checked" 
+            @change="check($event)"
+            class="mdc-checkbox__native-control"
+            id="checkbox-1"/>
+          <div class="mdc-checkbox__background">
+            <svg class="mdc-checkbox__checkmark"
+                viewBox="0 0 24 24">
```

Comment:
the above two lines can be written as:
```
<svg
  class="mdc-checkbox__checkmark"
  viewBox="0 0 24 24">
```
since you have more than one attribute in the HTML tag then every attribute can be written in a new line.